source/runner/DangerUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function sentence(array: Array<string>): string {
```

Comment:
Yes please, specifically so we can easily parse out the interfaces to make d.ts and flow defs